.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+/express/sitemap-index.xml @JingleH @fullcolorcoder @nateyolles @yeibercano
+/helix-sitemap.yaml @JingleH @fullcolorcoder @nateyolles @yeibercano
+/helix-query.yaml @JingleH @fullcolorcoder @nateyolles @yeibercano
+/fstab.yaml @JingleH @fullcolorcoder @nateyolles @yeibercano
+/head.html @JingleH @fullcolorcoder @nateyolles @yeibercano
```

Comment:
do we need an extra line end here?

Reply:
it's not required by github, but I think as a custom from li/unix world, configuration files usually end with a \n to indicate end of file